core/src/main/java/tc/oc/pgm/command/ScoreLimitCommand.java

```diff
@@ -0,0 +1,42 @@
+package tc.oc.pgm.command;
+
+import static net.kyori.adventure.text.Component.translatable;
+
+import app.ashcon.intake.Command;
+import app.ashcon.intake.parametric.annotation.Switch;
+import javax.annotation.Nullable;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.format.NamedTextColor;
+import tc.oc.pgm.api.Permissions;
+import tc.oc.pgm.api.match.Match;
+import tc.oc.pgm.score.ScoreMatchModule;
+import tc.oc.pgm.util.Audience;
+
+public final class ScoreLimitCommand {
+
+  @Command(
+      aliases = {"scorelimit", "sl"},
+      desc = "Set the score limit",
+      usage = "[number]",
+      help = "Score must be an integer",
+      flags = "r",
+      perms = Permissions.GAMEPLAY)
+  public void scorelimit(
+      Audience audience, Match match, @Nullable Integer amount, @Switch('r') boolean reset) {
+    final ScoreMatchModule score = match.getModule(ScoreMatchModule.class);
+    if (score != null) {
+      if (amount != null) {
+        score.setScoreLimit(amount);
+      } else if (reset) {
+        score.resetScoreLimit();
+      }
```

Comment:
```suggestion
      score.setScoreLimit(reset ? 0 : amount);
```

If you do the earlier check, amount should always be non-null if reset is false. Setting ScoreLimit to 0 should be equivalent to a reset, there's no need for a separate method

Reply:
Some maps have a score limit and not necessarily 0 points. E.g. KOTH has a set amount of points to obtain.